.eslintrc.json

```diff
@@ -129,7 +129,14 @@
     "promise/no-promise-in-callback": "off",
     "promise/no-return-in-finally": "error",
     "promise/valid-params": "error",
-    "vue/component-name-in-template-casing": ["error", "kebab-case"],
+    "vue/component-name-in-template-casing": ["error", "PascalCase", {
+      "registeredComponentsOnly": false,
+      "ignores": [
+        "vue-form",
+        "validate",
+        "field-messages"
```

Comment:
Why are these excluded?

Reply:
They currently can only be used in camelCase or kebab-case because they are registered in camelCase. We can either configure the names manually, or exclude them here. I opened a PR to change the default component names to PascalCase, which allows them to be used in all casings: https://github.com/fergaldoyle/vue-form/pull/128